adala/skills/collection/label_studio.py

```diff
@@ -53,7 +53,11 @@ def ner_tags(self) -> List[ControlTag]:
         for tag_name in control_tag_names:
             tag = self.label_interface.get_control(tag_name)
             if tag.tag.lower() in {"labels", "hypertextlabels"}:
-                tags.append(tag)
+                if self.allowed_object_tags:
```

Comment:
I'm trying to remember why we don't just always pass allowed_object_tags... do we have a good reason that this shouldn't always just be set, or have we kinda just stumbled into having to support both of these cases?
(I know we're not gonna fix it as part of this PR if that's the case, just want to make sure I understand here)

Reply:
https://github.com/HumanSignal/label-studio-enterprise/blob/develop/label_studio_enterprise/lse_ml_models/skills.py#L475

This is the reason I guess. I dont know the actual details of how passing it would drop information, since we only save filtered field_schema and keep the entire label_config anyway